docs/cases/cases-overview.asciidoc

```diff
@@ -102,3 +102,45 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
   
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
+
+[float]
+[[cases-export-import]]
+== Export and import cases
+
+Cases can be <<cases-export, exported>> and <<cases-import, imported>> as saved objects through the Kibana {kibana-ref}/managing-saved-objects.html[Saved Objects] UI.
+
+IMPORTANT: Before importing a Lens visualization into a space, ensure the visualization's data is present. Without it, the Lens visualization won't work.
+
+[float]
+[[cases-export]]
+=== Export a case
+Exporting cases can help you back up your cases or move them from one space to another. When you export a case, the case details, user actions, and comments are exported to a delimited JSON (ndjson) file. During the export process, text string comments, case alerts, and lens visualizations are exported.
+
+NOTE: Exported alerts will only contain indices' names, the alert document ID, and the ID and name of the rule that triggered the alert. Lens visualizations are exported as a JSON blob.
+
+To export a case:
+
+. Open the main menu, then click *Stack Management > Kibana > Saved Objects*.
+. Search for the case by choosing a saved object type or by entering the case title in the search bar.
+. Select one or more case and click the *Export* button.
+. Click *Export*. A toast message verifying that your file is downloading in the background displays.
+
++
+TIP: Keep the *Include related objects* option enabled to ensure connectors are also exported.
+
+[role="screenshot"]
+image::images/cases-export-button.png[Shows the export saved objects workflow]
+
+[float]
+[[cases-import]]
+=== Import a case
+
+To import a case:
+
+. Open the main menu, then click *Stack Management -> Kibana -> Saved Objects*.
+. Click *Import*.
+. Select the delimited JSON (ndjson) file containing the exported case and configure the import options.
```

Comment:
Instead of "delimited JSON", I would either say "Newline Delimited JSON" or just "ndjson" / "NDJSON"